keras/utils/np_utils.py

```diff
@@ -48,3 +49,62 @@ def normalize(x, axis=-1, order=2):
     l2 = np.atleast_1d(np.linalg.norm(x, order, axis))
     l2[l2 == 0] = 1
     return x / np.expand_dims(l2, axis)
+
+def mxnet_image_channels_conversion(np_data):
+    """
+    Transform the image tensor data from channels_last to channels_first
+    if using MXNet backend
+
+    # Arguments
+        np_data: image data tensor
+
+    # Returns
+        A image data tensor with channels_first format
+    """
+    data_format = K.image_data_format()
+    if data_format not in {'channels_first', 'channels_last'}:
+        raise ValueError('MXNet Backend: Unknown data_format ' +
+                         str(data_format))
+    if K.backend() == 'mxnet':
+        if data_format == 'channels_first':
+            shape = np_data.shape
```

Comment:
Handle both single and list of images.

Reply:
Done with help of `to_channels_first()` and `to_channels_first_helper()`